core/runtime/src/main/java/io/quarkus/runtime/ApplicationLifecycleManager.java

```diff
@@ -86,6 +87,9 @@ private ApplicationLifecycleManager() {
 
     private static final boolean IS_WINDOWS = System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("windows");
     private static final boolean IS_MAC = System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("mac");
+    // Only register the Quarkus thread dumper handler when the GraalVM one is unavailable.
+    private static final boolean shouldRegisterHandler = ImageMode.current() == ImageMode.NATIVE_BUILD
+            && GraalVM.Version.getCurrent().compareTo(GraalVM.Version.VERSION_24_2_0) < 0;
```

Comment:
Same here. Why compare to 24.2 and not 24.0?

Reply:
Personally, 24.0 seemed more intuitive to me, but I also understand the reasoning of only doing it for the version which also has potentially the conflicting `jcmd` feature enabled. Either way we go, please add a comment to that effect.